roles/openshift_health_checker/openshift_checks/__init__.py

```diff
@@ -97,12 +97,39 @@ def get_var(self, *keys, **kwargs):
         OpenShiftCheckException when a key is not found or returning a default value
         provided as a keyword argument.
         """
+        if len(keys) == 1:
+            keys = keys[0].split(".")
+
         try:
             value = reduce(operator.getitem, keys, self.task_vars)
         except (KeyError, TypeError):
             if "default" in kwargs:
                 return kwargs["default"]
             raise OpenShiftCheckException("'{}' is undefined".format(".".join(map(str, keys))))
+
+        convert = kwargs.get("type")
```

Comment:
bool("false") is True. Ansible inventory variables typically come in as strings, except where we've set them with facts already. This has already caused multiple bugs. See the change in logging/logging.py - someone who set `openshift_hosted_deploy_logging=False` in their inventory would see the checks expecting logging to be deployed. It's a really easy mistake to make.